src/graceful_termination.jl

```diff
@@ -61,6 +64,34 @@ container hook. Typically, K8s initiates graceful termination via the `TERM` sig
 as Julia forcefully terminates when receiving this signal and Julia does not support
 user-defined signal handlers we utilize `preStop` instead.
 
+### Using superviser entrypoint
+
+K8sDeputy provides a bash script in bin/superviser.sh which handles the `TERM` signal and
+sends the `"terminate"` message to the graceful termination socket.  This can be used as the
+`ENTRYPOINT` for your docker image after installation as follows (assuming an Ubuntu/Debian
+base image):
+
+```dockerfile
+# Copy Project/Manifest and instantiate to install K8sDeputy and other dependencies...
+
+# Install K8sDeputy termination shim and dependencies (netcat and jq)
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y netcat-openbsd jq && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    julia --color=yes -e 'using K8sDeputy; K8sDeputy.install_supervise_shim("/usr/bin")'
+
+ENTRYPOINT ["/usr/bin/supervise.sh"]
+```
+
+The command/script to run and any arguments it requires should be passed in via `args` in
+your [k8s Container
```

Comment:
Officially either k8s or K8s is fine. I've been using K8s in the documention here
```suggestion
your [K8s Container
```